src/Lykke.Service.HFT/Modules/ServiceModule.cs

```diff
@@ -128,6 +128,10 @@ private void BindRabbitMq(ContainerBuilder builder, AppSettings.HighFrequencyTra
                 .SingleInstance().AutoActivate();
             builder.RegisterType<ApiKeysConsumer>()
                 .WithParameter(TypedParameter.From(settings.ApiKeysFeed))
+                .WithParameter(
+                    new ResolvedParameter(
+                        (pi, ctx) => pi.ParameterType == typeof(IDistributedCache),
+                        (pi, ctx) => ctx.ResolveKeyed<IDistributedCache>("apiKeys")))
```

Comment:
Probably it can be simplified.
A Redis registration example
services.AddDistributedRedisCache(options =>
            {
                options.Configuration = RedisConfiguration;
                options.InstanceName = InstanceName;
            });

Reply:
I have different cache instances. One for api-keys, another for orderbooks. Services should use named cache instances.